### PR TITLE
fix: ie and edge drag

### PR DESCRIPTION
**Pull request by simboonlong**
- to degrade gracefully for both ie and edge
- will be showing button to click on directly for sample, instead of drag drop

### CODE REVIEW

**Comment by Nebulis** [comment]
ie or edge ?

**Comment by simboonlong** [comment]
oh was trying out, didn't work

File: src/components/HomePageContent/DropZoneSection.tsx [hunk 1]
```diff
@@ -51,16 +50,36 @@ const MobileDemoCertificate: React.FunctionComponent = () => (
...
+    this.state = {
+      isIeOrEdge: false,
+    };
   }
   componentDidMount(): void {
+    const browser = Bowser.getParser(window.navigator.userAgent);
+    // https://github.com/lancedikson/bowser/blob/master/src/constants.js
+    const isValidBrowser = browser.satisfies({
+      windows: {
+        chrome: "~87",
```
Comment (Nebulis) [inline src/components/HomePageContent/DropZoneSection.tsx:74]: does that means that people using google chrome on windows will see the button ?

Comment (simboonlong) [reply src/components/HomePageContent/DropZoneSection.tsx:74]: sadly yeah, not too good a solution eh?

Comment (Nebulis) [reply src/components/HomePageContent/DropZoneSection.tsx:74]: The lib doesn't support it? 

https://stackoverflow.com/questions/56360225/how-to-detect-microsoft-chromium-edge-chredge-edgium-in-javascript

Comment (simboonlong) [reply src/components/HomePageContent/DropZoneSection.tsx]: i thought there was an existing issue with the lib -> bowser/issues/387, as loading the demo on SOE itself was reflecting `Chrome` for both ie and edge browsers

decided to change the lib, both ie and edge browsers are still reflecting as `Chrome` when implemented. But if say i manually pasted the library into browser console and print out browser name it reflects as `IE` and `Edge` correctly... very strange

Comment (simboonlong) [reply src/components/HomePageContent/DropZoneSection.tsx]: ok i think SOE is not a good place to debug, the simple check from stackoverflow also printed out `chrome`

**Comment by simboonlong** [comment]
needs to be verified on IE or Edge (not on SOE) if solution's working as expected

**Comment by simboonlong** [comment]
stale, closing

**Pull request closed without merging**